src/core_plugins/kibana/public/dashboard/panel/dashboard_panel.test.js

```diff
@@ -26,6 +29,7 @@ function getProps(props = {}) {
 
 beforeAll(() => {
   store.dispatch(updateViewMode(DashboardViewMode.EDIT));
+  store.dispatch(setPanels([{ panelIndex: 'foo1' }]));
```

Comment:
it's setting up redux so that the dashboard will be rendered with a single panel with a panel id/index of `foo1` . `foo1` could be anything as long as it matches the defaultTestProps panel above.  Without this call it'll break because you will be passing in props that say render panel `foo1` but that panel won't exist in the store, so I'll get an error like ` Cannot read property 'title' of undefined`.

Reply:
sorry, I did not see this was a test file. Disregard the questions